sycl/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ option(SYCL_ADD_DEV_VERSION_POSTFIX "Adds -V postfix to version string" ON)
 set(SYCL_MAJOR_VERSION 1)
 set(SYCL_MINOR_VERSION 0)
 set(SYCL_PATCH_VERSION 0)
-set(SYCL_DEV_ABI_VERSION 1)
+set(SYCL_DEV_ABI_VERSION 2)
```

Comment:
From https://github.com/intel/llvm/blob/sycl/sycl/doc/ABIPolicyGuide.md:
> The development version of the library has a postfix -V that indicates breaking changes between releases.

@s-kanaev you didn't break ABI. Adding a new exported symbol is backwards-compatible change.